tests/examples/example-wrapped-data.nex

```diff
@@ -0,0 +1,30 @@
+#NEXUS
+
+begin data;
+	Dimensions ntax=3 nchar=471;
+	Format datatype=dna Missing=N gap=-;
+
+	matrix
+
+Boophis_ankaratra_Mandraka_old
+TCACTTGTTCTTTAAATGAGGACTAGAATCAACGGCATCACGAGGATCTTACTGTCTCCCCCCTCTAATCAGTGAAACTAATCTCCCCGTGAAGAAGCGG
+GGATAAAAATATAAGACGAGAAGACCCCATGGAGCTTTAAACCCAATAACACCCACTTATTTTACATCTTTAAATT-TACAAGGACATGTTTATTTGGTT
+TTAGGTTGGGGTGACCACGGAGTATAAAATAACCTCCATGAAAAAAAGATTAATAATCTTTATCAGAGAAATACACTTCCAAGAATTAATAAATTAATTT
+TTAATGATCCGATATTC----GATCAACGAACCAAGTTACCCTGGGGATAACAGCGCAATCCACCTCAAGAGCTCCTATCGACAGGTGGGTTTACGACCT
+CGATGTTGGATCAGGGTATCCCAGTGGTGCAGCAGCTACTAATGGTTCGTTTGTTCAACGATTTAAACCCT
+Boophis_septentrionalis_Masoala_98_FA_b11
+TCACTTGTTCTTTAAATGAGGACTAGAATCAACGGCATCACGAGGACCCCACTGTCTCCCCCCTCCAATCAGTGAAACTGATCTCTCCGTGAAGAAGCGG
+AGATATAAATATAAGACGAGAAGACCCCATGGAGCTTTAAACCCAACAACACCCATTTTATTTTTACCTATATATTAACTA-GGATTTGTTTGTTTGGTT
+TTAGGTTGGGGTGACCACGGAGCACAAAATAACCTCCATGAAAAAAGGACTCAAAATCCTTATCAAAGAAGCACACTTCTAAGAATTAATAAATTAATTT
+TTAATGATCCGATATTC----GATCAACGAACCAAGTTACCCTGGGGATAACAGCGCAATCCACCTTAAGAGCTCCTATCGACAGGTGGGTTTACGACCT
+CGATGTTGGATCAGGGTATCCTAGTGGTGCAGCAGCTACTAATGGTTCGTTTGTTCAACGATTAAAACCCT
+Boophis_septentrionalis_Ilampy_FA_b7
+TCACTTGTTCTTTAAATGAGGACTAGAATCAACGGCATCACGAGGACCCCACTGTCTCCCCCCTCCAATCAGTGAAACTGATCTCTCCGTGAAGAAGCGG
+AGATATAAATATAAGACGAGAAGACCCCATGGAGCTTTAAACCCAACAACACCCATTTTATTTTTACCCATATATTAACTA-GGATTTGTTTGTTTGGTT
+TTAGGTTGGGGTGACCACGGAGCACAAAATAACCTCCATGAAAAAAGGACTCAAAATCCTTATCAAAGAAGCACACTTCTAAGAATTAATAAATTAATTT
+TTAATGATCCGATATTC----GATCAACGAACCAAGTTACCCTGGGGATAACAGCGCAATCCACCTTAAGAGCTCCTATCGACAGGNNNNNNNNNNNNNN
+NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
+
+;
+end;
+

```

Comment:
@SimonGreenhill I guess, the question is whether we want to commit to such a new feature based on this example. OTOH, it seemed straightforward to implement (once one accepts the ugliness of iterating twice over the data block lines).

Reply:
Hmm, I've never seen that format before! Nexus is a joy that keeps giving. I have seen this style, which should work:

```
taxon1 AAA
taxon2 AAA
taxon3 AAA
taxon1 AAA
taxon2 AAA
taxon3 AAA
taxon1 AAA
taxon2 AAA
taxon3 AAA
```

so I guess this is a variant of that. Thanks for the fix.

Reply:
If I never have to write custom nexus parsing code again, I'll be happy. Let's check this in